build-system/tasks/presubmit-checks.js

```diff
@@ -654,6 +657,20 @@ const forbiddenTerms = {
       'src/inabox/amp-inabox-lite.js',
     ],
   },
+  'window\.document': {
```

Comment:
Simple string match linter is insufficient here. E.g. in #19511 the problematic snippet was `this.win.document`.

Have you tried doing this in conformance-config.textproto?